src/jquery.smooth-scroll.js

```diff
@@ -212,12 +212,9 @@
     // automatically calculate the speed of the scroll based on distance / coefficient
     if (speed === 'auto') {
 
-      // $scroller.scrollTop() is position before scroll, aniProps[scrollDir] is position after
+      // $scroller[scrollDir]() is position before scroll, aniProps[scrollDir] is position after
       // When delta is greater, speed will be greater.
-      delta = aniProps[scrollDir] - $scroller.scrollTop();
-      if(delta < 0) {
-        delta *= -1;
-      }
+      delta = Math.abs(aniProps[scrollDir] - $scroller[scrollDir]());
```

Comment:
Yeah, that reasoning makes sense. My intuition at the time, based on the benchmark, was that `Math.abs` is a method call off of a global, so the lookup from nested scope and subsequent call are probably slower, especially considering the comparison might allow us to skip the `*=` altogether. Again, though, I'm fine with the change. I'm not nearly as concerned with such micro-optimizations as I once was since they rarely provide any noticeable benefit.